test/gtest/uct/ib/test_ib.cc

```diff
@@ -102,6 +102,35 @@ class test_uct_ib_addr : public test_uct_ib {
 
         free(ib_addr);
     }
+
+    void test_fill_ah_attr(uint64_t subnet_prefix) {
+        uct_ib_iface_t *iface     = ucs_derived_of(m_e1->iface(), uct_ib_iface_t);
+        static const uint16_t lid = 0x1ee7;
+        union ibv_gid gid;
+        struct ibv_ah_attr ah_attr;
+
+        gid.global.subnet_prefix = subnet_prefix ?: iface->gid.global.subnet_prefix;
+        gid.global.interface_id  = 0xdeadbeef;
+
+        if ((iface->gid.global.subnet_prefix != UCT_IB_LINK_LOCAL_PREFIX) &&
+            (gid.global.subnet_prefix == UCT_IB_LINK_LOCAL_PREFIX)) {
+            UCS_TEST_SKIP_R("not applicable");
+        }
+
+        uct_ib_iface_fill_ah_attr_from_gid_lid(iface, lid, &gid, &ah_attr);
+
+        if (uct_ib_iface_is_roce(iface)) {
+            EXPECT_EQ(ah_attr.is_global, 1);
+        }
+
+        if (iface->is_global_addr) {
+            EXPECT_EQ(ah_attr.is_global, 1);
```

Comment:
in EXPECT_EQ macro, the expected value is 1st arg and actual value is 2nd arg

Reply:
that is why I used EXPECT_TRUE :)
fixed